cmd/update-rke2-charts/Makefile

```diff
@@ -0,0 +1,21 @@
+GO = go
+
+BINDIR := bin
+BINARY := update-rke2-charts
+
+VERSION = v0.1.0
+GIT_SHA = $(shell git rev-parse HEAD)
+LDFLAGS = -ldflags "-X main.gitSHA=$(GIT_SHA) -X main.version=$(VERSION) -X main.name=$(BINARY)"
+TAGS    = "netgo osusergo no_stage static_build"
+
+$(BINDIR)/$(BINARY): clean
+	CGO_ENABLED=1 $(GO) build -tags $(TAGS) -v $(LDFLAGS) -o $@
+
+.PHONY: clean
+clean:
+	$(GO) clean
+	rm -f $(BINDIR)/$(BINARY)
+
+.PHONY: image
+image: $(BINDIR)/$(BINARY)
+	docker build -t rancher/$(BINARY):$(VERSION) .
```

Comment:
I think this is a fair question, which I don't have the answer to. I simply imitated the file layout I found for the other Go tools on this repository. But now I think of it, it might be no longer needed wdyt @briandowns ?

Reply:
This appears to be vestigial. We're going to be invoking this via a container but not directly for this app. This bit can be removed from the Makefile.